kaspresso/kaspresso-framework/src/main/kotlin/com/kaspersky/kaspresso/internal/outscreens/NotificationScreen.kt

```diff
@@ -0,0 +1,13 @@
+package com.kaspersky.kaspresso.internal.outscreens
+
+import com.kaspersky.components.kautomator.component.switch.UiSwitch
+import com.kaspersky.components.kautomator.screen.UiScreen
+
+internal object NotificationScreen : UiScreen<NotificationScreen>() {
+
+    override val packageName: String = "com.android.systemui"
+
+    val wifiSwitcher = UiSwitch {
+        withContentDescription("Wi-Fi,Wifi signal full.,AndroidWifi")
```

Comment:
Does it work with real devices?

Reply:
Good question. I am going to research

Reply:
@RuslanMingaliev Nope, it so depends on a type of a device. On each device, the flow is different. I suggest describing these restrictions in the documentation of `Network` class and the throwing exception. What do you think?

Reply:
It doesn't work even with emulator, I tried Android 10 x86. 

> I suggest describing these restrictions in the documentation of Network class and the throwing exception.

I think any restrictions are evil. We should avoid them in any way.